core/compose/compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.7'
+services:
+  blueos-core:
+    image: bluerobotics/blueos-core:1.2.0-beta.6
+    container_name: blueos-core
+    privileged: true
+    # This can be used as an alternative to ports binding
+    # Using host as a network mode is only supported for linux
+    # Ref: https://docs.docker.com/network/drivers/host
+    # network_mode: host
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /run/udev:/run/udev
+      - /etc/resolv.conf:/etc/resolv.conf
+      - /etc/machine-id:/etc/machine-id
+      - ./workspace/userdata:/usr/blueos/userdata
+      - ./workspace/logs:/var/logs/blueos
+      - ./workspace/config:/root/.config
+      - ./workspace/blueos:/etc/blueos
+      # Allow changes on repository reflects directly on the running container
+      - ../start-blueos-core:/usr/bin/start-blueos-core
+      - ../services:/home/pi/services
+    pid: "host"
+    environment:
+      - BLUEOS_DISABLE_SERVICES=cable_guy,wifi,commander
+      - BLUEOS_DISABLE_MEMORY_LIMIT=true
+    ports:
```

Comment:
Shouldn't the websockets also be here?

Couldn't we just use `network_mode: "host"` instead?

Reply:
The websockets use the same ports as the apis, no ? At least the is the case for mavlink2rest.

there is a comment explaining about network_mode host on the top.

Reply:
the MCM uses 6021 for the websocket

Reply:
@joaoantoniocardoso added

Reply:
Should we add the RTSP server too? (8554)

Reply:
added

Reply:
I'm not seeing it :x

Reply:
Sorry! I put the code and forget to push it. Should be there now